### ISSUE TITLE
Panel Link Usability Regression with new Stat Panel in 7.x

### ISSUE THREAD

**Issue by fkellner** [grafana/grafana#26805]
In my Dashboards that I created in version 6.7, I used small SingelStat Panels with Panel Links as "Buttons with Info", basically they show a summary of the Drilldown Dashboard that they link to by changing color.
I found it particularly great that on hover, the cursor would turn into a pointer and show the link title, making it immediately clear to users that _this is a button that leads me somewhere_, even in kiosk mode.
See old singleStat panel:
![image](https://user-images.githubusercontent.com/28141677/89412406-505a1a00-d727-11ea-937e-e9d73f78756c.png)

With the new stats panel, panel links and data links do not turn the cursor into a pointer on hover. So there is basically no way for a user to find out something is a link without knowing they can click it.

Panel links are also invisible in kiosk mode, which makes kind of sense since it is called "kiosk" mode, but I really rather used it as "non-editing, distraction-free mode" for users to browse a set of interlinked dashboards in.

Take this Stat panel with a data link AND a panel link in kiosk mode as an example:
![image](https://user-images.githubusercontent.com/28141677/89412757-e55d1300-d727-11ea-838d-8fa1e88a4817.png)

**What would you like to be added**:
If there is no special hover behaviour for a panel, no data links and only one panel link, please restore the old behaviour.

**Why is this needed**:
How else should users know that they can click stuff?

**Comment by Rushmed** [comment]
Hi, is there any intension to solve this and make the previous setting with goto possible?

**Comment by fkellner** [comment]
Thank you for warning me that the Singlestat panel links will break!

I found a workaround using the [Dynamic Text Plugin](https://grafana.com/grafana/plugins/marcusolsson-dynamictext-panel).
It is rather ugly and much worse to maintain than a Singlestat panel, but it replicates the desired behaviour and will allow me to upgrade without breaking my dashboards.
Paste into "Content" field and change stuff to your liking:
```handlebars
{{!-- this replaces the deprecated singlestat panel with panel link as good as possible --}}
<a style="display: block; text-align: center;" 
{{!-- set PANEL LINK URL below --}}
href="https://url-to-your-grafana-instance.com/d/VbDQPqTZz/errors-drilldown?orgId=1&refresh=1m&kiosk"
{{!-- set PANEL LINK TITLE below --}}
title="Look at errors in detail">
  <h2 style="display:inline-block; margin-right: 0.5em; color:
{{!-- set THRESHOLDS and COLORS below --}}
{{#if (gt count 1)}}rgb(196, 22, 42){{else if (gt count 0)}}color:rgba(237, 129, 40, 0.89){{else}}rgb(55, 135, 45){{/if}}
;">
{{!-- METRIC PREFIX below --}}
errors:
</h2>
  <h1 style="display:inline-block; color:
{{!-- set THRESHOLDS and COLORS below --}}
{{#if (gt count 1)}}rgb(196, 22, 42){{else if (gt count 0)}}color:rgba(237, 129, 40, 0.89){{else}}rgb(55, 135, 45){{/if}}
;">
{{!-- insert METRIC VARIABLE NAME below --}}
{{count}}
</h1>
</a>
```
As you can see, styling needs to be done as attribute, since the plugin does not allow the addition of `<style>` nodes (and we need to override the default styling). Also, thresholds are done via handlebars. 
If anyone has a nicer solution (or the developers could bring back panel links), I would be very happy.

**Comment by Rushmed** [comment]
Nice job but I can`t understand why this "onboard" feature is gone. Still hope that it will be back in the upcomming updtaes!

**Comment by dprokop** [comment]
@Rushmed @fkellner this feature is not gone but changed. You can use data links to achieve similar behavior. In 7.0 we distinguished the concept of [data links](https://grafana.com/docs/grafana/latest/linking/data-links/) and [panel links](https://grafana.com/docs/grafana/latest/linking/panel-links/). 

The data links are shown in a context menu though (cursor icon indicates this), so the difference there is the user needs to first open the context menu(click on the viz) to reach the data link.

**Comment by fkellner** [comment]
As I described in my original post, panel links do not adequately replace the old behaviour:
- context menu is invisible in kiosk mode
- not reachable simply by clicking on a panel
- cursor does not become pointer on hovering the panel (in version 7.3, did not update to 7.4 yet because see above). 

I basically used the old singlestat panels to create "buttons" that link to other dashboards while showing some summary of them. The traffic that this issue gets tells me that I am not the only one doing this.

tl;dr:There IS a big difference between a big fat button and a teeny tiny small i in the upper left corner.

**Comment by dprokop** [comment]
> tl;dr:There IS a big difference between a big fat button and a teeny tiny small i in the upper left corner.

I agree with your point. I won't make any promises about ETA, but we'll talk this through and reconsider the behaviour of the data link in the Stat panel.

**Comment by Rushmed** [comment]
If there is only one panel link added the panel link should open directly by clicking.

**Comment by maxblaze** [comment]
_Please_ make this option available again, this is a big deal for us. Usability was _much_ better before–the tiny buttons are really difficult to click in an emergency.

**Comment by torkelo** [comment]
@Rushmed make it a data link instead and it behaves like that

**Comment by maxblaze** [comment]
So I have a Stats panel with a single AWS CloudWatch metric. Clicking the panel takes me to the AWS web console and there doesn't seem to be a way to override it with either a panel or data link.

**Comment by torkelo** [comment]
@maxblaze if you define the data link in an override it should override the link coming from the cloudwatch data source
